Firmware/UI/Painter/Painter.cpp

```diff
@@ -2,6 +2,7 @@
 
 #include <cstring>
 #include <cstdlib>
+#include <iostream>
```

Comment:
Please avoid <iostream> in firmware, as it blows up the size of it.

Reply:
Okay, should I remove and commit again?

Reply:
Nope, just remove the line and commit the fix. PR should update automatically, at least normally.